metaci/release/models.py

```diff
@@ -17,6 +17,28 @@ def __str__(self):
         return self.name
 
 
+class ImplementationStep(models.Model):
+    description = models.ForeignKey(
+        "plan.Plan", on_delete=models.CASCADE, related_name="name"
+    )
+    release = models.ForeignKey(
+        "release.Release", on_delete=models.CASCADE, related_name="releases"
+    )
+    plan = models.ForeignKey(
+        "plan.Plan", on_delete=models.CASCADE, related_name="plans"
+    )
+    start_time = models.DateField(
+        _("start_time"),
+        null=True,
+        blank=True,
+        default=None,
+    )
+    stop_time = models.DateField(_("stop_time"), null=True, blank=True, default=None)
```

Comment:
These should be DateTimeField so we can also set a time, and they should not have null=True or blank=True (implementation steps need to be planned for a specific window of time)